packages/wallets/src/translations/i18n.ts

```diff
@@ -48,6 +49,14 @@ const resources = {
     },
 };
 
+function generateKey(inputString: string) {
```

Comment:
any reason for the function being duplicated, not included here? 
if I understand correctly, they both needs to be strictly in sync to make it work, so might be good to avoid risk of them getting out of sync

Reply:
yeaa I forgot to extract these out into a separate utility function, will do just that so its only one source of truth in the utils folder